include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -295,12 +295,20 @@ typedef struct OrtTensorRTProviderOptions {
   int device_id;                                // cuda device id.
   int has_user_compute_stream;                  // indicator of user specified CUDA compute stream.
   void* user_compute_stream;                    // user specified CUDA compute stream.
-  int has_trt_options;                          // override environment variables with following TensorRT settings at runtime.
```

Comment:
I was wondering can we remove field here?
This api is been exposed and what if a user is using SessionOptionsAppendExecutionProvider_TensorRT() and directly using this struct?
Can user's app work with our latest ORT? since the ABI is not preserved.

Reply:
the field was added after last release, so technically it wasn't part of an official release yet.

Reply:
Got it. That won't be a problem.